app/Export/ImageConversion.hs

```diff
@@ -22,9 +25,9 @@ convertToImage :: String -> String -> IO
                       Maybe Handle,
                       ProcessHandle)
 convertToImage inName outName = createProcess $ CreateProcess
-                                  (ShellCommand $ "convert " ++
+                                  (ShellCommand $ "rsvg-convert " ++
```

Comment:
@AilsaF Please revert this change. I removed it specifically because it did not work for me.